### DISCUSSION TITLE
The domain suraj.is-a.dev has been verified by another user/organization

### DISCUSSION THREAD

**Discussion by Clay990** [is-a-dev/register#6858] [Support] [answered]
![Screenshot (522)](https://github.com/is-a-dev/register/assets/83020021/db88ebed-7812-411d-b7ed-ddcd89a75086)
how to solve this verify issue? i don't have much knowledge about github!

**Comment by wdhdev** [comment, marked as answer]
Please refer to this issue: #6627
